torchvision/models/squeezenet.py

```diff
@@ -4,10 +4,8 @@
 import torch.nn.init as init
 import torch.utils.model_zoo as model_zoo
 
-
```

Comment:
can you revert those unnecessary line changes and space changes in this file?

Reply:
OK